### ISSUE TITLE
`fdk_format_drivers` works for rsofun pmodel, but not rsofun BiomeE model.

### ISSUE THREAD

**Issue by fabern** [geco-bern/FluxDataKit#103]
In order to use FluxDataKit with the BiomeE part of the `rsofun` package some additional input quantities need to be defined.
These are documented in the rsofun package: https://geco-bern.github.io/rsofun/reference/biomee_p_model_drivers.html

- To make it work with pmodel, some default parameters need to be prescribed for `params_siml`:
    https://github.com/geco-bern/FluxDataKit/blob/e0223b39d585e978d2cef2b875cbb9e82146bba5/R/fdk_format_drivers.R#L27-L39

- [ ] To make it work with BiomE-model in a similar way, some additional parameters would need to be provided, too:
    - `site_info` containing:
       - `lon`, `lat`, `evl`, `year_start`, `year_end`, `classid`, `c4`, `whc`, `koeppen_code`, `igbp_land_use`, `plant_functional_type`, `date_start`, and `date_end` 
    - `params_siml` containing:
       - `spinup`, `spinupyears`, `recycle`, `firstyeartrend`, `nyeartrend`, `steps_per_day`, `do_U_shaped_mortality`, `update_annualLAImax`, `do_closedN_run`, `method_photosynth`, `method_mortality`
    - `params_tile` containing:
       - `soiltype`, `FLDCAP`, `WILTPT`, `K1`, `K2`, `K_nitrogen`, `MLmixRatio`, `etaN`, `LMAmin`, `fsc_fine`, `fsc_wood`, `GR_factor`, `l_fract`, `retransN`, `f_initialBSW`, `f_N_add`, `tf_base`, `par_mort`, `par_mort_under`
    - `params_species` containing:
       - `lifeform`, `phenotype`, `pt`, `alpha_FR`, `rho_FR`, `root_r`, `root_zeta`, `Kw_root`, `leaf_size`, `Vmax`, `Vannual`, `wet_leaf_dreg`, `m_cond`, `alpha_phot`, `gamma_L`, `gamma_LN`, `gamma_SW`, `gamma_FR`, `tc_crit`, `tc_crit_on`, `gdd_crit`, `betaON`, `betaOFF`, `alphaHT`, `thetaHT`, `alphaCA`, `thetaCA`, `alphaBM`, `thetaBM`, `seedlingsize`, `maturalage`, `v_seed`, `mortrate_d_c`, `mortrate_d_u`, `LMA`, `leafLS`, `LNbase`, `CNleafsupport`, `rho_wood`, `taperfactor`, `lAImax`, `tauNSC`, `fNSNmax`, `phiCSA`, `CNleaf0`, `CNsw0`, `CNwood0`, `CNroot0`, `CNseed0`, `Nfixrate0`, `NfixCost0`, `internal_gap_frac`, `kphio`, `phiRL`, `LAI_light`
    - `params_soil` containing:
       - `type`, `GMD`, `GSD`, `vwc_sat`, `chb`, `psi_sat_ref`, `k_sat_ref`, `alphaSoil`, `heat_capacity_dry`
    - `init_cohort` containing:
       - `init_n_cohorts`, `init_cohort_species`, `init_cohort_nindivs`, `init_cohort_bl`, `init_cohort_br`, `init_cohort_bsw`, `init_cohort_bHW`, `init_cohort_seedC`, `init_cohort_nsc`
    - `init_soil` containing:
       - `init_fast_soil_C`, `init_slow_soil_C`, `init_Nmineral`, `N_input`

**Comment by stineb** [comment]
Is this still unresolved by recent modifications to homogenize forcings? @fabern

**Comment by fabern** [comment]
Yes this is still an open issue. I modified the original post to clarify that it is about defining default parametrization of BiomeE to be able to create a runnable driver.
